Gem/Settings/Navigation/NetworkSelectorNavigationStack.swift

```diff
@@ -1,35 +1,84 @@
 // Copyright (c). Gem Wallet. All rights reserved.
 
 import SwiftUI
+import Settings
 import Primitives
 import Components
+import Style
 
 struct NetworkSelectorNavigationStack: View {
-    
     let chains: [Chain]
-    @State var selectedChain: Binding<Chain>
-    @State var isPresenting: Binding<Bool>
+    @Binding var selectedChain: Chain
+    @Binding var isPresenting: Bool
+
+    @State private var searchQuery = ""
+
     var action: ((String) -> Void)?
 
+    var filteredChains: [Chain] {
```

Comment:
can you move this into view model?

Reply:
Yeah, I would like to) , but there is no way to add vm here without changing a structure of project. If we want to include vm there, we have to rewrite AddTokenNavigationStack, AddTokenScene.